test/parallel/test-runner-output.mjs

```diff
@@ -203,6 +203,10 @@ const tests = [
     name: 'test-runner/output/arbitrary-output.js',
     flags: ['--test-reporter=tap'],
   },
+  {
+    name: 'test-runner/output/non-tty-forced-color-output.js',
+    flags: ['--test-reporter=spec'],
+  },
```

Comment:
Instead of testing the test runner, add a test where a non-TTY FORCE_COLOR is used.

This change isn't specific to the test runner

Reply:
IMO it's not breaking, it's a bug fix